autoload/lsp/omni.vim

```diff
@@ -288,6 +303,13 @@ function! lsp#omni#get_vim_completion_items(options) abort
         let l:incomplete = 0
     endif
 
+    let l:sort = has_key(l:server, 'config') && has_key(l:server['config'], 'sort') ? l:server['config']['sort'] : v:none
```

Comment:
`v:none` does not exist on Neovim. `v:null` is available instead.

Reply:
Oh.. I did not know about this.
Use `v:null` instead!